Tests/MessagingInApp/Views/InAppMessageViewTest.swift

```diff
@@ -84,7 +80,8 @@ class InAppMessageViewTest: UnitTest {
 
         let inlineView = InAppMessageView(elementId: .random)
 
-        XCTAssertFalse(isDisplayingInAppMessage(inlineView))
+        XCTAssertFalse(isInlineViewVisible(inlineView))
+        XCTAssertNil(getInAppMessage(forView: inlineView)) // expect not in process of rendering a message
```

Comment:
It's important to test not only if a View is visible but also check if the View is currently in the process of rendering a message. 

I improved this test function to check both of these states.